docs/deployment/cli.md

```diff
@@ -23,664 +23,254 @@ specific language governing permissions and limitations
-                           execution/deployment and for the configured executor.
-                           The available options can be found at
-                           https://ci.apache.org/projects/flink/flink-docs-stabl
-                           e/deployment/config.html
-     -e,--executor <arg>   DEPRECATED: Please use the -t option instead which is
-                           also available with the "Application Mode".
-                           The name of the executor to be used for executing the
-                           given job, which is equivalent to the
-                           "execution.target" config option. The currently
-                           available executors are: "remote", "local",
-                           "kubernetes-session", "yarn-per-job", "yarn-session".
-     -t,--target <arg>     The deployment target for the given application,
-                           which is equivalent to the "execution.target" config
-                           option. The currently available targets are:
-                           "remote", "local", "kubernetes-session",
-                           "yarn-per-job", "yarn-session", "yarn-application"
-                           and "kubernetes-application".
-
-  Options for yarn-cluster mode:
-     -m,--jobmanager <arg>            Set to yarn-cluster to use YARN execution
-                                      mode.
-     -yid,--yarnapplicationId <arg>   Attach to running YARN session
-     -z,--zookeeperNamespace <arg>    Namespace to create the Zookeeper
-                                      sub-paths for high availability mode
-
-  Options for default mode:
-     -m,--jobmanager <arg>           Address of the JobManager to which to
-                                     connect. Use this flag to connect to a
-                                     different JobManager than the one specified
-                                     in the configuration. Attention: This
-                                     option is respected only if the
-                                     high-availability configuration is NONE.
-     -z,--zookeeperNamespace <arg>   Namespace to create the Zookeeper sub-paths
-                                     for high availability mode
-{% endhighlight %}
+## Advanced CLI
+ 
+### REST API
+
+The Flink cluster can be also managed using the [REST API]({% link ops/rest_api.md %}). The commands 
+described in previous sections are a subset of what is offered by Flink's REST endpoints. Therefore, 
+tools like `curl` can be used to get even more out of Flink.
+
+### Selecting Deployment Targets
+
+Flink supports different deployment backends like [Kubernetes]({% link deployment/resource-providers/native_kubernetes.md %}) 
+or [YARN]({% link deployment/resource-providers/yarn.md %}) which are described in more detail in the 
+Resource Provider section. You can specify the mode you're Flink cluster is running in through the 
```

Comment:
Link to the Resource Provider Section?

Reply:
We don't have a dedicated resource provider section. That's why I mentioned two examples to be able to at least reference to their subsections.